### PR TITLE
Make documentation version ready

### LINKED ISSUE #36: Make documentation version ready
We will have more than one version and we need to keep all released documentation (except SNAPSHOTs where only the last one matters).

### PR DESCRIPTION
**Pull request by FaustXVI**
Closes #36 
- [X] Generate a new documentation for each version
- [x] Create link to newest version

### CODE REVIEW

File: .travis/publish-documentation.sh [hunk 1]
```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
```
Comment (vdemeester) [inline .travis/publish-documentation.sh]: `#!/usr/bin/env bash` for nix ? :angel:

**Review by vdemeester — APPROVED** [review]
Just a nit/question but LGTM :frog:

**Comment by FaustXVI** [comment]
Will it work on any linux?

Xavier Detant

**Comment by vdemeester** [comment]
@FaustXVI yes, more than `/bin/bash` :angel:

**Review by vdemeester — APPROVED** [review]
LGTM 🐯

**Pull request merged**